android/src/com/mapswithme/maps/widget/recycler/DividerItemDecorationWithPadding.java

```diff
@@ -25,7 +26,7 @@ public class DividerItemDecorationWithPadding extends RecyclerView.ItemDecoratio
 
   public DividerItemDecorationWithPadding(@NonNull Context context)
   {
-    mDivider = Objects.requireNonNull(context.getDrawable(R.drawable.divider_base));
+    mDivider = Objects.requireNonNull(AppCompatResources.getDrawable(context, R.drawable.divider_base));
```

Comment:
Blood from my eyes :) assert(mDivider != null); is much better IMHO than Objects.requireNonNull(...)
Lint requires it?

Reply:
`AppCompatResources.getDrawable()` is nullable. I see that `assert x` are not popular in Android code base and we throw AssertionErrors manually.

Reply:
OMG, no, do not bloat the code, keep requireNonNull if there is no other options.

My opinion, if assert(mDivider != null) suppresses lint error, better to use it. If not - ok, use requireNonNull.

Where do we throw AssertionErrors manually? What is the difference with compact single assert expression? Release will crash anyway in both cases.

Reply:
> Where do we throw AssertionErrors manually? 

https://github.com/organicmaps/organicmaps/search?q=AssertionError&type=code

> What is the difference with compact single assert expression? Release will crash anyway in both cases.

AFAIK, `assert` statement is disabled by default on Android until you set  `adb shell setprop debug.assert`. Explicit `throw new AssertionError` will work.